agenta-web/src/components/Evaluations/Evaluations.tsx

```diff
@@ -8,6 +19,7 @@ import EvaluationsList from "./EvaluationsList"
 import {EvaluationFlow, EvaluationType} from "@/lib/enums"
 import {EvaluationTypeLabels} from "@/lib/helpers/utils"
 import Title from "antd/es/typography/Title"
+import Text from "antd/es/typography/Text"
```

Comment:
This is the source of the error we previously had. Please check https://github.com/Agenta-AI/agenta/pull/162 for the fix!